src/pseudopeople/interface.py

```diff
@@ -407,23 +386,61 @@ def generate_social_security(
 
 
 def fetch_filepaths(dataset, source):
+    # todo: add typing
     if dataset.name == DatasetNames.TAXES_1040:
         tax_dataset_names = [
             DatasetNames.TAXES_1040,
             DatasetNames.TAXES_W2_1099,
             DatasetNames.TAXES_DEPENDENTS,
         ]
-        data_paths = {}
         for tax_dataset in tax_dataset_names:
             directory = source / tax_dataset
             dataset_paths = [x for x in directory.glob(f"{tax_dataset}*")]
-            data_paths[tax_dataset] = dataset_paths
+            sorted_dataset_paths = sorted(dataset_paths)
+            if tax_dataset == DatasetNames.TAXES_1040:
+                data_paths = [{} for i in range(len(sorted_dataset_paths))]
```

Comment:
If you define this `data_paths` object outside the loop, you don't need the if statement.